openapi/data/fields.py

```diff
@@ -109,8 +109,8 @@ def date_field(**kw):
     return data_field(**kw)
 
 
-def date_time_field(**kw):
-    kw.setdefault('validator', DateTimeValidator())
+def date_time_field(timezone=False, **kw):
```

Comment:
Can we call this something like `tz_aware`? Otherwise I expect some timezone object.

Reply:
``sqlalchemy`` uses that timezone=True